source-map-uploader-plugin.js

```diff
@@ -54,7 +54,8 @@ class BugsnagSourceMapUploaderPlugin {
           url: resolve(
             // ensure publicPath has a trailing slash
             publicPath.replace(/[^/]$/, '$&/'),
-            source
+            // ensure source doesn't have a leading slash
```

Comment:
it might be good to have this comment describe why this is being done, instead of what is being done